### PR TITLE
added a few more configurable options

### PR DESCRIPTION
**Pull request by daniel-vlad**
Hi,

I've added a few more configurable options for the placeholder. Needed the component in one of our projects and having the ability to configure a few more things would have been a nice feature.
If you think this is ok and merge worthy please do so 😄 

Daniel

### CODE REVIEW

**Review by ankon — CHANGES_REQUESTED** [review]
The changes contain a lot of unneeded whitespace changes/reindentation, could you please undo these?

**Comment by daniel-vlad** [comment]
will do. sorry about that, looks like vscode autoformat messed things up a bit

**Comment by daniel-vlad** [comment]
@ankon , Changed the indentation so that it matches the original one and added missing eof new lines

File: animated-content-placeholder.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 ```html
 <animated-content-placeholder>
-    <div class="background-masker header-left"></div>
-    <div class="background-masker header-right"></div>
+        <div class="background-masker header-left"></div>
+        <div class="background-masker header-right"></div>
 </animated-content-placeholder>
```
Comment (ankon) [inline animated-content-placeholder.html]: Hmm. 

This doesn't look right. 

Note that we're using actual tab characters, and do not replace tabs with any particular amount of spaces.

**Review by ankon — CHANGES_REQUESTED** [review]

**Pull request merged**